cl/favorites/views.py

```diff
@@ -173,3 +174,17 @@ async def view_tags(request, username):
             "private": False,
         },
     )
+
+
+async def open_prayers(request: HttpRequest) -> HttpResponse:
```

Comment:
Can we cache this page for at least 30s, say?

Reply:
Done in [8a36f56](https://github.com/freelawproject/courtlistener/pull/4386/commits/8a36f56065f3a4587b35ddec11c3e6bcce39f178)